Library/Formula/glib.rb

```diff
@@ -3,6 +3,7 @@ class Glib < Formula
   homepage "https://developer.gnome.org/glib/"
   url "https://download.gnome.org/sources/glib/2.46/glib-2.46.1.tar.xz"
   sha256 "5a1f03b952ebc3a7e9f612b8724f70898183e31503db329b4f15d07163c8fdfb"
+  revision 1
```

Comment:
The bottles only work on this formula for `/usr/local` Homebrew users, and for 10.9-10.11. With a `revision` you'd be forcing every non-/usr/local and everyone on <10.9 to rebuild `glib` from source for a non-essential issue. We try to limit how often that sort of thing happens, because it's a pain.